### DISCUSSION TITLE
Help for hosting website

### DISCUSSION THREAD

**Discussion by piyushyadav0191** [vercel/next.js#25232] [Help]
I have submit my files on repositories and they are committed but when i go to my username and github dot io.my website isn't opening and showing 404 error. How can i host my website and i have no idea please help

**Comment by adithyapaib** [comment]
u need a live server to build and deploy nextjs apps
try vercel.com for nextjs apps deployment

**Reply by piyushyadav0191** [reply]
Thank you😄
